### PR TITLE
Added map component to the results

### PR DESCRIPTION
**Pull request by cw00dw0rd**
The map features uses:
- [Leaflet](https://leafletjs.com/) to render and control
- [Mapbox](https://www.mapbox.com/) for the map imaging

Opted for displaying results to the left and having the map always visible. `Results` is a big enough component that it may be worthwhile to break it up into smaller ones.

### CODE REVIEW

**Comment by cw00dw0rd** [comment]
I like the event log and I think it will make sense in the long term to have a hideable third column to the left of results, that shows this along with #31

**Comment by cw00dw0rd** [comment]
I started splitting up some components that were getting large and added in the first GEO query to get results for listings in an area. These listings should display on the map with the newly created arangoIcon. 

A larger change was the addition of Vuex but it was becoming necessary to have state management.

File: jsconfig.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
     "include": [
-      "./workspaces/frontend-vue/src/**/*"
+        "./workspaces/frontend-vue/src/**/*"
+    ],
+    "exclude": [
+        "node_modules",
+        "./workspaces/frontend-vue/node_modules"
```
Comment (couds) [inline jsconfig.json:7]: maybe we can add this jsconfig inside `workspaces/frontend-vue` instead of in the root? because this is for the vue project only

**Comment by couds** [comment]
Can we merge the api part? So we can also work the react frontend in parallel =)

**Comment by cw00dw0rd** [comment]
> Can we merge the api part? So we can also work the react frontend in parallel =)

Merged 😄 sorry for delay, the PR went on a little long.

**Pull request merged**